consensus/istanbul/ibft/core/core.go

```diff
@@ -98,9 +98,9 @@ func (c *core) finalizeMessage(msg *ibfttypes.Message) ([]byte, error) {
 	msg.Address = c.Address()
 
 	// Add proof of consensus
```

Comment:
is this comment valid here or needs to be moved?

Reply:
Moved it.